io.openems.edge.battery.bmw/src/io/openems/edge/battery/bmw/Config.java

```diff
@@ -0,0 +1,55 @@
+package io.openems.edge.battery.bmw;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition( //
+		name = "BMW Battery", //
```

Comment:
Use something like `BMS BMW` to have it sorted properly in Apache Felix (see https://github.com/OpenEMS/openems/blob/develop/io.openems.edge.battery.soltaro/src/io/openems/edge/battery/soltaro/single/versionc/Config.java#L10)